ext/NamedGraphsSimpleGraphAlgorithmsExt/NamedGraphsSimpleGraphAlgorithmsExt.jl

```diff
@@ -0,0 +1,17 @@
+module NamedGraphsSimpleGraphAlgorithmsExt
+using Graphs: AbstractGraph, edgetype
+using NamedGraphs: position_graph, vertices
+using NamedGraphs.GraphsExtensions: GraphsExtensions
+using SimpleGraphAlgorithms: SimpleGraphAlgorithms
+using SimpleGraphConverter: UG
+
+function SimpleGraphAlgorithms.edge_color(g::AbstractGraph, k::Int64)
```

Comment:
```suggestion
function SimpleGraphAlgorithms.edge_color(g::AbstractNamedGraph, k::Int64)
```
Otherwise this would be [type piracy](https://docs.julialang.org/en/v1/manual/style-guide/#avoid-type-piracy).